docs/optimizations/FileLocalization.md

```diff
@@ -42,11 +53,10 @@ task nio_task {
 
 This optimization is currently only applied to localization in the Pipelines API (GCE) backends.
 
-## Warning
+## Portability Warning
 
 Be very careful not to write unportable WDL files!
```

Comment:
TOL: try to avoid double-negatives. Which is to say, write in the positive tone.

"Be very careful to write portable WDL files", rather than "Be very careful **not** to write **un**portable WDL files"